### ISSUE TITLE
question about freezing layers in ResNet backbone

### ISSUE THREAD

**Issue by sanghoooon** [serend1p1ty/SeqNet#10]
Hi. Thank you for great research and an awesome implementation.

It's not really an issue, but I have a minor question about your model implementation in models/resnet.py.
Here, you freeze conv1/batchnorm layers within ResNet, as follows:
```
resnet.conv1.weight.requires_grad_(False)    
resnet.bn1.weight.requires_grad_(False)    
resnet.bn1.bias.requires_grad_(False)
```

Is there a particular reason behind this choice?

**Comment by serend1p1ty** [comment]
Hi, it is a common practice when finetuning the model pretrained in ImageNet.
This trick is common used in other projects, such as https://github.com/facebookresearch/detectron2/blob/df6eac1814e7182625640d9168e4308bd3a6a8f8/detectron2/modeling/backbone/resnet.py#L468

The amount of data in finetuning is usually relatively small, and freezing the stem of ResNet is conducive to stability.

**Comment by sanghoooon** [comment]
Okay, looks like a common thing to do.
Thank you for your reply.
